src/drivers/pca9685_pwm_out/PCA9685.cpp

```diff
@@ -162,7 +156,7 @@ void PCA9685::setPWM(uint8_t channel, const uint16_t &value)
 	int ret = transfer(buf, 5, nullptr, 0);
 
 	if (OK != ret) {
-		PX4_DEBUG("i2c::transfer returned %d", ret);
+		PX4_ERR("setPWM: i2c::transfer returned %d", ret);
```

Comment:
Do you need these in general? They can be a bit dangerous if you hit a real persistent error and then start spamming the console. On a small embedded system trying to print to the console at high rate can disrupt the scheduling of other sensors. Maybe only an annoyance on a more powerful LInux system.

Reply:
Thanks for pointing out that. I will keep only the ones from init.